### DISCUSSION TITLE
Retroarch scalefx shader for ikemen

### DISCUSSION THREAD

**Discussion by TIGRE2410** [ikemen-engine/Ikemen-GO#1156] [Feature Request]
Hi, I would like ikemen to be able to use the scalefx shader from retroarch, it's the best shader I've seen...
i think scalefx is .glsl and ikemen only supports .frag and .vert in external/shaders folder

You can see how good this shader is in the following videos:

https://www.youtube.com/watch?v=m05L3mNQ58M&ab_channel=RetroCrisis
https://www.youtube.com/watch?v=3AzBusZfsxA&ab_channel=Sp00kyFox
https://www.youtube.com/watch?v=kxLJkUtIaHk&ab_channel=Sp00kyFox
https://www.youtube.com/watch?v=gx0XkEZ65-g&ab_channel=Sp00kyFox

Why do I want this shader for ikemen?
because there are many sprites that are very good but they are very different, either in size or in shading, and when used in the same game they look horrible. Examples: tmnt sprites vs blaz blue, these sprites are quite different and would combine very well if this shader is used in ikemen
Another example would be facing HD sprites with kof or cvs style sprites, although from a distance they look moderately decent, we all know that they clash a lot when the characters are very close to the screen
Another example: face the sprites of sain seiya HD with those of jojo bizarre, with this shader the sprites would seem taken from the same game and the difference in the size of the pixels (always annoying) would end and we could use those sprites for the same quality game
I am not a hater of pixel art, otherwise I would not be in mugen or ikemen, I want the shader simply to unify the characters that look very different and be able to use them in the same game without feeling that they are taken out of context. It is something that the shader HQ2X and HQ4X did not achieve

**Comment by Orochikyocr** [comment]
I wanted to add new shaders to Ikemen Go But since Ikemen Go is stuck in OpenGL 2.0 so people with old computers can use it, most shaders I tried to use did not work.
